homeassistant/components/hp_ilo/switch.py

```diff
@@ -0,0 +1,87 @@
+"""Support for powering on/off server with HP iLO."""
+import logging
+
+import hpilo
```

Comment:
Only import the login function and maybe rename it to something more self-explanatory.